setup.py

```diff
@@ -6,20 +6,21 @@
 
 setup(
   name = 'chewBBACA',
-  packages = ['CHEWBBACA','CHEWBBACA.allelecall','CHEWBBACA.createschema','CHEWBBACA.utils','CHEWBBACA.SchemaEvaluator','CHEWBBACA.PrepExternalSchema'],
+  packages = ['CHEWBBACA', 'CHEWBBACA.allelecall', 'CHEWBBACA.createschema',
+              'CHEWBBACA.utils','CHEWBBACA.SchemaEvaluator','CHEWBBACA.PrepExternalSchema',
+              'CHEWBBACA.CHEWBBACA_NS'],
   version = VERSION,
   description = 'A complete suite for gene-by-gene schema creation and strain identification',
-  author = 'Mickael Silva',
-  author_email = 'mickaelsilva@medicina.ulisboa.pt',
+  author = 'Mickael Silva, Pedro Cerqueira, Rafael Mamede',
```

Comment:
Mickael's email is not listed, move him to the back of the line?

Reply:
Changed `author_email` value to group email. Professional email of any of the authors will not be used anymore.